src/controllers/user.controllers.js

```diff
@@ -29,16 +30,17 @@ export const userLogin = async (data) => {
   const { error } = loginValidation(data);
   if (error) return { status: 'invalid', message: error.details[0].message };
 
-  const user = await User.findOne({ email: data.email });
+  // const user = await User.findOne({ email: data.email });
+  const user = await User.findOne({ email: 'Patryk123@gmail.com' });
   if (!user) return { status: 'invalid', message: 'Email or password is wrong' };
-
-  const validPass = await bcrypt.compare(data.password, user.password);
+  // const validPass = await bcrypt.compare(data.password, user.password);
+  const validPass = await bcrypt.compare('test123', user.password);
   if (!validPass) return { status: 'invalid', message: 'Email or password is wrong' };
 
+  // Zapytać o to
   const token = jsonwebtoken.sign({ _id: user._id }, process.env.TOKEN_SECRET);
-  const header = ('auth-token', token);
 
-  return `Witam ${user.name}!`;
+  return { message: { token, message: `Witam ${user.name}!` } };
```

Comment:
You could simplify return to 
```suggestion
  return { token, message: `Welcome ${user.name}!` };
```